### PR TITLE
[fix](mtmv)fix insert overwrite getExecTimeout error

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

should use InsertTimeout but QueryTimeout
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java [hunk 1]
```diff
@@ -402,6 +403,7 @@ public boolean isSyncLoadKindStmt() {
         if (parsedStmt instanceof LogicalPlanAdapter) {
             LogicalPlan logicalPlan = ((LogicalPlanAdapter) parsedStmt).getLogicalPlan();
             return logicalPlan instanceof InsertIntoTableCommand
+                    || logicalPlan instanceof InsertOverwriteTableCommand
```
Comment (TangSiyang2001) [inline fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java:406]: should also fix InsertOverwriteTableStmt for legacy planner

**Review by Hastyshell — CHANGES_REQUESTED** [review]

**Comment by zddr** [comment]
run buildall

**Review by Hastyshell — APPROVED** [review]
LGTM

**Comment by zddr** [comment]
run pipelinex_p0

**Comment by zddr** [comment]
run pipelinex_p0

**Comment by zddr** [comment]
run pipelinex_p0

**Review by morrySnow — APPROVED** [review]

**Pull request merged**